apps/accounts/templates/base.html

```diff
@@ -7,6 +7,62 @@
 		<meta charset="UTF-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		{%- comment -%} Load sentry before any CSS/other JS loads {%- endcomment -%}
+		
+
+<script
+  src="https://js.sentry-cdn.com/dd30eb65f140420688f813ab96721a1c.min.js"
+  crossorigin="anonymous"
+></script>
+
+<script>
+  window.sentryOnLoad = function () {
+    Sentry.init({
+      dsn: "https://dd30eb65f140420688f813ab96721a1c@o14742.ingest.sentry.io/2071988",
```

Comment:
I think this would result in every environment (staging/ prod etc) we have reporting to the same dashboard in sentry. I'd prefer this was variable passed in if possible, but I appreciate it's a thing I'm probably better placed to add, as we'd like rely on this being an environment variable or similar.

Reply:
Cheers Chris. Guidance here in the Sentry docs for that:
https://docs.sentry.io/platforms/javascript/configuration/environments/